<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sportsed::Client::Printer::MainWindow</class>
 <widget class="Sportsed::Client::ClientMainWindow" name="Sportsed::Client::Printer::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Printer Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabSettings">
       <attribute name="title">
        <string>Printer Settings</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Printer:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
          <item>
           <widget class="QComboBox" name="printerBox"/>
          </item>
          <item>
           <widget class="QPushButton" name="refreshPrintersBtn">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Location:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="locationLbl">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="descriptionLbl">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Model:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="modelLbl">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="pageSetupBtn">
          <property name="text">
           <string>Page Setup</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabReports">
       <attribute name="title">
        <string>Reports</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <item>
         <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,0">
          <item row="0" column="0">
           <widget class="QComboBox" name="reportTypeBox">
            <item>
             <property name="text">
              <string>Start - By class</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Start - By time</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Results - Preliminary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Results - Official</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Multi-stage Results - Preliminary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Muti-stage Results - Official</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="printNowBtn">
            <property name="text">
             <string>Print Now</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="startBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>88</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_7" columnstretch="1,0">
          <item row="0" column="0" rowspan="2">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Classes</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0" colspan="2">
              <widget class="QListView" name="classView"/>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="classesTogetherBtn">
               <property name="text">
                <string>Together</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QRadioButton" name="classesSeparateBtn">
               <property name="text">
                <string>Separate</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>288</width>
              <height>138</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Add Automation</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <widget class="QRadioButton" name="newAutomationEveryBtn">
               <property name="text">
                <string>Every</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QSpinBox" name="newAutomationEveryEdit">
               <property name="suffix">
                <string> min</string>
               </property>
               <property name="singleStep">
                <number>5</number>
               </property>
               <property name="value">
                <number>15</number>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>(if changed)</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QRadioButton" name="newAutomationOnCompleteBtn">
               <property name="text">
                <string>On class complete</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2" rowspan="2" colspan="3">
              <widget class="QGroupBox" name="groupBox_6">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="1" column="1">
                 <widget class="QSpinBox" name="newAutomationWaitEdit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="suffix">
                   <string> min</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QTimeEdit" name="newAutomationAfterEdit">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QCheckBox" name="newAutomationAfterBtn">
                  <property name="text">
                   <string>After</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="newAutomationWaitBtn">
                  <property name="text">
                   <string>Wait</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="4">
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>13</width>
                 <height>21</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="3" column="0" colspan="3">
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>198</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="3" column="3" colspan="2">
              <widget class="QPushButton" name="newAutomationAddBtn">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabAutomation">
       <attribute name="title">
        <string>Automations</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QListView" name="automationsView"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Information</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="automationDescription">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>240</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="deleteAutomationBtn">
             <property name="text">
              <string>Delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <addaction name="actionLoadProfile"/>
     <addaction name="actionStoreProfile"/>
    </widget>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="menuProfile"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>&amp;Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
  </action>
  <action name="actionLoadProfile">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="actionStoreProfile">
   <property name="text">
    <string>&amp;Store</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Sportsed::Client::ClientMainWindow</class>
   <extends>QMainWindow</extends>
   <header>clientlib/ClientMainWindow.h</header>
   <container>1</container>
   <slots>
    <slot>quit()</slot>
    <slot>about()</slot>
    <slot>aboutQt()</slot>
    <slot>connectClicked()</slot>
    <slot>disconnectClicked()</slot>
    <slot>loadProfileClicked()</slot>
    <slot>storeProfileClicked()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>newAutomationEveryBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>newAutomationEveryEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newAutomationOnCompleteBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>612</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newAutomationAfterBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>newAutomationAfterEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>613</x>
     <y>430</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newAutomationWaitBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>newAutomationWaitEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>698</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newAutomationEveryBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_6</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
